source/_posts/2017-02-13-automation-deployment-with-apache-kafka.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "Automation Deployment with Apache Kafka"
+author: Bryan Baugher
+date: 2017-02-13
+tags: [engineering, kafka]
+---
+
+Its likely not a surprise that Cerner would use Apache Kafka as we have used
+a number of related technologies like [Apache Hadoop](http://engineering.cerner.com/2013/02/composable-mapreduce-with-hadoop-and-crunch/)
+along with its Map/Reduce, HDFS and even [Apache HBase](http://engineering.cerner.com/2013/02/near-real-time-processing-over-hadoop-and-hbase/).
+Our team first started using [Apache Kafka](http://kafka.apache.org/) in 2014
+when Kafka 0.8 first came out. Since then we've expanded to using Kafka for a
+number of different use cases and it has become a core piece of Cerner's infrastructure.
```

Comment:
Optional blog/videos we can link to for expanding on our use cases 
http://blog.cloudera.com/blog/2014/11/how-cerner-uses-cdh-with-apache-kafka/
https://www.confluent.io/kafka-summit-2016-users-ingesting-complex-healthcare-data-with-apache-kafka